packages/forma-36-website/src/pages/components/radio-button/index.mdx

```diff
@@ -5,17 +5,18 @@ status: 'stable'
 github: 'https://github.com/contentful/forma-36/tree/master/packages/forma-36-react-components/src/components/RadioButton'
 storybook: 'https://f36-storybook.contentful.com/?path=/story/components-radiobutton--default'
 ---
+
 Detailed documentation will be provided soon. For now, please have a look into the storybook for implementation details.
 
 ```jsx
 <>
   <RadioButtonField
-    labelText='Radio button label'
+    labelText="Radio button label"
     helpText="This is a helptext"
     name="someOption"
-    checked="true"
+    checked={true}
```

Comment:
What do you think about going one step further and do this?

```jsx
<RadioButtonField
  // ...
  checked
```

Reply:
Since the value is true, we only need to pass a value when it's false

Reply:
The very related checkbox for example has `checked={true}` in the example instead of just `checked`.
https://f36.contentful.com/components/checkbox/

That's why I didn't shorten it.

Reply:
Ok cool! No problem

we can shorten all of them later if we want, maybe :)